tests/fixtures/base-fixtures.ts

```
@@ -0,0 +1,217 @@
+  attachTestSummary(result: Partial<TestResult>): Promise<void>;
+  logTestStart(message: string, data?: any): void;
+  logTestEnd(message: string, data?: any): void;
+  logInfo(message: string, data?: any): void;
+  logError(message: string, data?: any): void;
+  logDebug(message: string, data?: any): void;
+}
+
+export const baseTest = base.extend<BaseTestFixtures, BaseWorkerFixtures>({
+  // Configuration manager - worker scoped for better performance
+  configManager: [
+    async ({}, use: (config: ConfigurationManager) => Promise<void>) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)